R/recipe_run.R

```diff
@@ -514,30 +538,45 @@ orderly_prepare_data <- function(config, info, parameters, envir) {
   ## Compute the device stack size before starting work too
   n_dev <- length(grDevices::dev.list())
 
-  hash_resources <- hash_files(expand_directory_list(info$resources))
-  if (length(hash_resources) > 0L) {
-    orderly_log("resources",
-                sprintf("%s: %s", names(hash_resources), hash_resources))
-  } else {
-    hash_resources <- NULL
-  }
+  # hash_resources <- hash_files(expand_directory_list(info$resources))
```

Comment:
please remove the commented out sections. Your function is a nicer way of doing thi